includes/class-wc-countries.php

```diff
@@ -944,6 +944,16 @@ public function get_country_locale() {
 							'required' => false,
 						),
 					),
+					'GH' => array(
+						'postcode' => array(
+							'label'    => __( 'Postcode', 'woocommerce' ),
```

Comment:
You don't need to include a label if this is a hidden field.

Reply:
Ok. I just used the entry for Nigeria(NG) as a frame of reference for this, because I know the field gets hidden when you select Nigeria. Should I remove the label then?

Reply:
@claudiosanches I see what you mean, I'll edit this now